pkgs/by-name/ne/nexusmods-app/package.nix

```diff
@@ -1,87 +1,87 @@
-{ _7zz
-, buildDotnetModule
-, copyDesktopItems
-, desktop-file-utils
-, dotnetCorePackages
-, fetchFromGitHub
-, fontconfig
-, lib
-, libICE
-, libSM
-, libX11
-, nexusmods-app
-, runCommand
-, enableUnfree ? false # Set to true to support RAR format mods
+{
+  _7zz,
+  buildDotnetModule,
+  copyDesktopItems,
+  desktop-file-utils,
+  dotnetCorePackages,
+  fetchFromGitHub,
+  fontconfig,
+  lib,
+  libICE,
+  libSM,
+  libX11,
+  nexusmods-app,
+  runCommand,
+  enableUnfree ? false, # Set to true to support RAR format mods
 }:
-let
-  _7zzWithOptionalUnfreeRarSupport = _7zz.override {
-    inherit enableUnfree;
-  };
-in
 buildDotnetModule rec {
-  pname = "nexusmods-app";
+  pname = "nexusmods-app" + lib.optionalString enableUnfree "-unfree";
 
-  version = "0.4.1";
+  version = "0.5.2";
 
   src = fetchFromGitHub {
     owner = "Nexus-Mods";
     repo = "NexusMods.App";
     rev = "v${version}";
     fetchSubmodules = true;
-    hash = "sha256-FzQphMhiC1g+6qmk/R1v4rq2ldy35NcaWm0RR1UlwLA=";
+    hash = "sha256-NQUHvMGFeexy/EYvEwZrXUHAAuBqA3fyGm7FYjXTq6s=";
   };
 
-  projectFile = "NexusMods.App.sln";
+  projectFile = "src/NexusMods.App/NexusMods.App.csproj";
```

Comment:
I noticed that making this change stops all the `dotnet test` tests from running.  Perhaps this change can be removed once https://github.com/NixOS/nixpkgs/pull/327651 is in?